pubspec.yaml

```
@@ -3,21 +3,20 @@ version: 4.11.1
 description: A library for building statically-typed React UI components using Dart.
 homepage: https://github.com/Workiva/over_react/
 environment:
-  sdk: ">=2.11.0 <3.0.0"
+  sdk: '>=2.13.0 <3.0.0'
 
 dependencies:
-  collection: ^1.14.11
+  collection: ^1.15.0
   analyzer: ^5.1.0
   build: '>=1.0.0 <3.0.0'
-  built_redux: ^8.0.6
-  built_value: ^8.0.0
   dart_style: ^2.0.0
   js: ^0.6.1+1
-  logging: ^1.0.0
+  logging: '>=0.11.3+2 <2.0.0'
```

Looks like this was probably a bad merge from before this dep got updated in master; we need `^1.0.0` since it's the first null-safe release.
```suggestion
  logging: ^1.0.0
```